src/posthog-core.ts

```diff
@@ -1341,15 +1341,6 @@ export class PostHog {
         this.surveys.canRenderSurvey(surveyId)
     }
 
-    /** Get the next step of the survey: a question index or `end` */
-    getNextSurveyStep(
```

Comment:
since this is on posthog core could someone have integrated with it? should we keep the method and replace it instead with a deprecation log message, then replace it after some time?

(you all the experts on surveys so i may be worrying unnecessarily)

because of the lazy loading we always have to consider someone with posthog-js main bundle pre-this-change and latest lazy bundle

Reply:
should be fine, i checked for that! ran all e2e tests on posthog/posthog and did smoke tests for surveys extensively. didn't saw any other usage besides surveys' domain